testing/endtoend/endtoend_setup_test.go

```diff
@@ -115,16 +116,17 @@ func e2eMainnet(t *testing.T, usePrysmSh, useMultiClient bool, cfgo ...types.E2E
 		ev.PeersConnect,
 		ev.HealthzCheck,
 		ev.MetricsCheck,
-		ev.ValidatorsAreActive,
```

Comment:
Should this be removed?

Reply:
This is removed in favor of Kasey's new DepositedValidatorsAreActive evaluator.